madmom/audio/filters.py

```diff
@@ -1531,3 +1531,72 @@ def __array_finalize__(self, obj):
         # set default values here
         self.fref = getattr(obj, 'fref', A4)
         self.window = getattr(obj, 'window', self.WINDOW)
+
+
+class SemitoneBandpassFilterbank(object):
+    """
+    Time domain semitone filterbank of elliptic filters as propsed in [1]_.
+
+    Parameters
+    ----------
+    filter_order : int, optional
+        Filter order of elliptic filters.
+    passband_ripple_db : int, optional
+        Passband ripple in [Db]
```

Comment:
The abbreviation for Dezibel is `dB` not `Db`. I think `passband_ripple` is enough, since it is usually given in `dB` and this is also indicated in the description. Is there a reason why it should be `int` and not a `float`?